scripts/extension_helpers.js

```diff
@@ -2,6 +2,20 @@
 // Distributed under the terms of the Modified BSD License.
 
 var path = require('path');
+var findImports = require('find-imports');
+
+// Get the list of vendor files.
+var VENDOR_FILES = findImports('../lib/**/*.js', { flatten: true });
+var CODEMIRROR_FILES = VENDOR_FILES.filter(function(importPath) {
+  return importPath.indexOf('codemirror') !== -1;
+});
+var codemirrorPaths = CODEMIRROR_FILES.map(function(importPath) {
+  return importPath.replace('.js', '');
+});
+VENDOR_FILES = VENDOR_FILES.filter(function (importPath) {
+  return (importPath.indexOf('codemirror') === -1 && 
+          importPath.indexOf('phosphor') === -1);
```

Comment:
Same as above - is it enough to check that the importPath *starts* with codemirror or phosphor?